File: testsrc/ResourcePackerTest.scala
import java.io.File

import com.esotericsoftware.minlog.Log
import darkyenus.resourcepacker.{PackingOperation, LocalWorkingRootProvider, LWJGLLauncher}

/**
 *
 * @author Darkyen
 */
object ResourcePackerTest extends App {
  Log.DEBUG()
  LWJGLLauncher.launch(new PackingOperation(new File("testresources"),new File("target/RPTestResult"),workingRootProvider = new LocalWorkingRootProvider(new File("target/RPTestWorkingRoot"))))
}

File: src/test/java/ResourcePackerTest.kt
import com.darkyen.resourcepacker.LocalWorkingRootProvider
import com.darkyen.resourcepacker.packResources

import com.esotericsoftware.minlog.Log
import java.io.File

/**
 *
 */
fun main(args: Array<String>) {
    Log.DEBUG()
    packResources(File("src/test/resources"), File("build/cache/RPTestResult"),
                    workingRootProvider = LocalWorkingRootProvider("build/cache/RPTestWorkingRoot"))
}
